src/notation/internal/masternotation.cpp

```diff
@@ -482,76 +487,45 @@ INotationPartsPtr MasterNotation::parts() const
     return m_parts;
 }
 
-INotationPtr MasterNotation::clone() const
+ExcerptNotationList MasterNotation::availableExcerpts() const
```

Comment:
I think, better to rename to potentialExcerpts (this name is used in the UX document) or to possibleExcerpts